test/e2e_node/services/kubelet.go

```diff
@@ -90,10 +90,11 @@ func RunKubelet() {
 const (
 	// Ports of different e2e services.
 	kubeletReadOnlyPort = "10255"
```

Comment:
We can remove this `kubeletReadOnlyPort` because it is not used after replacing line 97 with `kubeletHealthzPort`.